### PR TITLE
D8UN-809

### PR DESCRIPTION
**Pull request by DuttonMa**
Drupal core 9.2.10 upgrade

### CODE REVIEW

**Comment by ooneill547** [comment]
Should we hold of on this for release today?

**Comment by DuttonMa** [comment]
> Should we hold of on this for release today?

Yes, good thought, we'll let it run in the overnight edge build before releasing it

**Review by ooneill547 — APPROVED** [review]

File: composer.lock [hunk 1]
```diff
@@ -1906,16 +1906,16 @@
         },
         {
             "name": "dof-dss/nicsdru_unity_theme",
-            "version": "v0.1.39",
+            "version": "v0.1.38",
```
Comment (ooneill547) [inline composer.lock:1909]: your local version of unity theme needs updated, this might cause issues.

Comment (DuttonMa) [reply composer.lock]: Great catch ! Have fixed that now

**Pull request merged**